contrib/fedora/rpm/NetworkManager.spec

```diff
@@ -237,6 +237,16 @@ Ethernet, Bridge, Bond, VLAN, Team, InfiniBand, Wi-Fi, mobile broadband
 services.
 
 
+%package dispatcher
+Summary: NetworkManager dispatcher
+Group: System Environment/Base
+Requires: %{name}%{?_isa} = %{epoch}:%{version}-%{release}
+
+%description dispatcher
+This package contains NetworkManager dispatcher. It provides support for
+running custom scripts upon various events signalled by NetworkManager.
```

Comment:
I am not against this patch. But it's not sufficiently clear to me whether this package split is a good idea or necessary.

Could you describe the advantages in the commit message more in depth? And why you think this is a good idea.

Note, that the dispatcher (systemd) service already can be masked. I think, `systemctl disable NetworkManager-dispatcher.service` should also suffice, as that would remove the D-Bus socket unit (didn't test that though).

Reply:
This is essentially a package-split. I think you need to make proper use of `Obsoletes` and `Provides`.

Reply:
I've now replaced the patch with one that merely adds a Provide that packages that install dispatcher scripts can depend on.

I intend to submit it some later point, taking your comments into account.